### DISCUSSION TITLE
Does the system recognize people who pass by but without being connected to any wifi network, but only with wifi on?

### DISCUSSION THREAD

**Discussion by HelloWar75** [cyberman54/ESP32-Paxcounter#726] [Q&A] [answered]
Does the system recognize people who pass by but without being connected to any wifi network, but only with wifi on?

**Comment by cyberman54** [comment, marked as answer]
Yes, of course, this is the main use case.

**Reply by HelloWar75** [reply]
So does he recognize? is that I did not understand the answer sorry.

**Reply by FlorianLudwig** [reply]
The PAXCounter counts devices not connected to a WiFi.
